### DISCUSSION TITLE
p/555e3035e0e24d22a98afbf526765705/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#34] [Announcements]
# p/555e3035e0e24d22a98afbf526765705/

自从 10 月初来日本之后，至今也快一个月了，现在可以谈谈更多的感受了。 城市-京都京都真的非常适合骑自行车，自从购买了自行车之后，无论是通勤还是出去玩，基本上都使用自行车，而京都这个城市足够小，出去玩使用自行车很方便。而且京都附近的一些地方的景色真的太美了，尤其是鸭川附近，真的随手都能拍出壁纸的感觉。  目前去了鸭川和伏见稻荷大社，两者的体验都还不错，而且都是免费的！让吾辈有点想知道，为什么大多

https://blog.rxliuli.com/p/555e3035e0e24d22a98afbf526765705/

**Comment by olojiang** [comment]
学习咋样啊？没有谈到学校的生活，除了一次聚餐（你先吃完，我再吃的习俗相当“秀丽”）

**Comment by rxliuli** [comment]
这和工作有点像，虽然占用的时间不少，但没有太多值得注意的。

**Comment by ethuh** [comment]
博主请问下你的网站会支持rss订阅吗，想关注你的网站

**Reply by rxliuli** [reply]
早就支持了啊
https://blog.rxliuli.com/atom.xml

**Reply by ethuh** [reply]
好的，已经订阅，加油
